lib/modules/versioning/ubuntu/index.ts

```diff
@@ -54,8 +66,31 @@ function isStable(version: string): boolean {
 
 // digestion of version
 
+function getDatedContainerImageCodename(version: string): null | string {
```

Comment:
Instead of having two different parsing function have one which returns an object. 

That way you have to parse only once in `equals`

Reply:
I would still need to parse twice if I use the common `getVersionByCodename` function as the rest of the `get` commands need it (major, minor etc).

I could re-implement the logic of `getVersionByCodename` in the equals function?